react-native/training/src/config/api.js

```diff
@@ -1,7 +1,9 @@
 import Reactotron from 'reactotron-react-native';
 import { create } from 'apisauce';
 
-const baseURL = 'http://wolox.com';
+import { isAndroid } from '../constants/platform';
+
+const baseURL = isAndroid ? 'http://10.0.2.2:3001' : 'http://localhost:3001';
```

Comment:
Put the api urls in the code is insecure. Define de api urls in .env file, and use here